docs/management/admin/deploy-with-mdm.asciidoc

```diff
@@ -0,0 +1,105 @@
+=== Enable notifications
+
+. Select the **Notifications** option to configure the Notification Center policy for the {elastic-endpoint} configuration profile. 
+. Under **App Name**, enter `Elastic Security.app`.
+. Under **Bundle ID**, enter `co.elastic.alert`.
+. In the **Settings** section:
+.. Include **Critical Alerts**, and select **Enable**.
+.. Include **Notifications**, and select **Enable**.
+.. Include **Banner alert type**, and set it to **Persistent**.
+.. Include **Notifications on Lock Screen**, and select **Display**.
+.. Include **Notifications in Notification Center**, and select **Display**.
+.. Include **Badge app icon**, and select **Display**.
+.. Include **Play sound for notifications**, and select **Enable**.
+. Save the configuration.
+
+[role="screenshot"]
+image::images/notifications-jamf.png[]
+
+[discrete]
+[[fda-jamf]]
+=== Enable Full Disk Access
+
+. Select the **Privacy Preferences Policy Control** option to configure the Full Disk Access policy for the {elastic-endpoint} configuration profile.
+. Add a new entry with the following details:
+.. Under **Identifier**, enter `co.elastic.systemextension`.
+.. From the **Identifier Type** dropdown, select **Bundle ID**.
+.. Under **Code Requirement**, enter the following:
+`identifier "co.elastic.systemextension" and anchor apple generic and certificate 1[field.1.2.840.113635.100.6.2.6] /* exists */ and certificate leaf[field.1.2.840.113635.100.6.1.13] /* exists */ and certificate leaf[subject.OU] = "2BT3HPN62Z"`
+.. Make sure that **Validate the Static Code Requirement** is selected.
+. Add a second entry with the following details:
+.. Under **Identifier**,  enter `/Library/Elastic/Endpoint/elastic-endpoint`.
+.. From the **Identifier Type** dropdown, select **Path**.
+.. Under **Code Requirement**, enter the following:
+`identifier "64_Bit_Endpoint_Macos" and anchor apple generic and certificate 1[field.1.2.840.113635.100.6.2.6] /* exists */ and certificate leaf[field.1.2.840.113635.100.6.1.13] /* exists */ and certificate leaf[subject.OU] = "2BT3HPN62Z"`
+.. Make sure that **Validate the Static Code Requirement** is selected.
+. Add a third entry with the following details:
+.. Under **Identifier**, enter `co.elastic.endpoint`.
+.. From the **Identifier Type** dropdown, select **Bundle ID**.
+.. Under **Code Requirement**, enter the following:
+`identifier "co.elastic.endpoint" and anchor apple generic and certificate 1[field.1.2.840.113635.100.6.2.6] /* exists */ and certificate leaf[field.1.2.840.113635.100.6.1.13] /* exists */ and certificate leaf[subject.OU] = "2BT3HPN62Z"`
+.. Make sure that **Validate the Static Code Requirement** is selected.
+. Save the configuration.
+
+[role="screenshot"]
+image::images/fda-jamf.png[]
+
+After you complete these steps, generate the mobile configuration profile and install it onto the macOS machines. Once the profile is installed, {elastic-defend} can be deployed without the need for user interaction.
```

Comment:
In this last step, would users click **Save** to generate the mobile configuration profile? If so, that might be worth adding in here.

Reply:
@caitlinbetz are there additional steps we need to document for 'generating the mobile configuration profile'? Or is the profile generated by configuring all the steps above?

Reply:
It sounds like this is a Jamf procedure for which Elastic has no special requirements, so we could leave it up to users to follow Jamf's docs on how to do this part. 🤷